scripts/HootEnv.sh

```diff
@@ -13,7 +13,7 @@ export CCACHE_SLOPPINESS=time_macros
 export GDAL_LIB_DIR=`gdal-config --libs | sed -e "s/-L//g" | sed -e "s/ *-lgdal.*//g"`
 export GDAL_DATA=`gdal-config --datadir`
 
-export LD_LIBRARY_PATH=$GDAL_LIB_DIR:$HOOT_HOME/lib:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=$GDAL_LIB_DIR:/usr/lib/jvm/java-1.8.0-openjdk/jre/lib/amd64/server:$HOOT_HOME/lib:$LD_LIBRARY_PATH
```

Comment:
Here we can use the variable `$JAVA_HOME` that is setup in a few different places.
```
VagrantProvisionCentOS7.sh:
echo "export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk" >> ~/.bash_profile
```

$ grep -r JAVA_HOME *
...
```suggestion
export LD_LIBRARY_PATH=$GDAL_LIB_DIR:$JAVA_HOME/jre/lib/amd64/server:$HOOT_HOME/lib:$LD_LIBRARY_PATH
```

Reply:
good call.